src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -61,6 +61,7 @@ public void IncorrectVariableThrowsArgument()
         }
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Environment Variables monidification isn't supported in Appx.")]
```

Comment:
@danmosemsft Yeah I understand. I think it makes sense to put the Win32Api method name in the comment so that we can easily grep it. I would prefer not putting a comment next to it but to use the reason parameter in xunit (like I did there).